<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VMEle</class>
 <widget class="QWidget" name="VMEle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>201</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>304</width>
    <height>195</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>308</width>
    <height>201</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="connectPushButton">
       <property name="minimumSize">
        <size>
         <width>308</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>308</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/image/win7-light.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="vmNameLabel">
       <property name="minimumSize">
        <size>
         <width>308</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>308</width>
         <height>15</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
qproperty-alignment: AlignCenter;
}</string>
       </property>
       <property name="text">
        <string>Windows1</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>308</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>308</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="connectsPushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="toolTip">
             <string>连接</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{border-image: url(:/image/image/connect-light.png)}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="startPushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="toolTip">
             <string>开机</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {border-image: url(:/image/image/start-light.png);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="restartPushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {border-image: url(:/image/image/restart-light.png);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="shutdownPushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="toolTip">
             <string>关机</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {border-image: url(:/image/image/shutdown-light.png);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
